xbmc/utils/FileUtils.cpp

```diff
@@ -261,12 +264,17 @@ bool CFileUtils::CheckFileAccessAllowed(const std::string &filePath)
     "/.ssh/",
   };
   // ALLOW kodi paths
-  const std::vector<std::string> whitelist = {
-    CSpecialProtocol::TranslatePath("special://home"),
-    CSpecialProtocol::TranslatePath("special://xbmc"),
-    CSpecialProtocol::TranslatePath("special://musicartistsinfo")
+  std::vector<std::string> whitelist = {
+      CSpecialProtocol::TranslatePath("special://home"),
+      CSpecialProtocol::TranslatePath("special://xbmc"),
+      CSpecialProtocol::TranslatePath("special://musicartistsinfo"),
```

Comment:
This indentation looks inconsistent to me (4 spaces instead of 2), but this is what @jenkins4kodi told me to do: https://github.com/xbmc/xbmc/pull/21113#issuecomment-1066397871